app/src/Pages/Home/Home.js

```diff
@@ -316,64 +339,113 @@ class HomePage extends React.Component {
         >
           {!!mapData.features.length && (
             <>
-              <BaseMap
-                updateViewport={false}
-                initialZoom={11}
-                useContainerHeight
+              <div
+                css={css`
+                  height: 700px;
+                  width: 100%;
+                `}
```

Comment:
It would be nice to abstract this as a class. That will also make the purpose of the `div` clearer for future eyes.